src/components/Notification/Profile.tsx

```diff
@@ -10,12 +11,14 @@ interface Props {
 }
 
 export const NotificationProfileAvatar: FC<Props> = ({ profile }) => {
+  const isNftAvatar = profile?.picture?.__typename === 'NftImage';
+
   return (
     <Link href={`/u/${profile?.handle}`}>
       <a href={`/u/${profile?.handle}`}>
         <img
           src={getAvatar(profile)}
-          className="w-8 h-8 bg-gray-200 rounded-full border dark:border-gray-700/80"
+          className={clsx('w-8 h-8', isNftAvatar ? 'hexagon' : 'rounded-full')}
```

Comment:
**suggestion:** I think it's better to have the border and we can create another class name in CSS file so it will not look long 🙇🏼